schema/me/append_conversation_thread.js

```diff
@@ -35,25 +47,31 @@ export default mutationWithClientMutationId({
     },
   },
   outputFields: {
-    message: {
-      type: MessagePayloadType,
-      resolve: message => message,
+    payload: {
+      type: AppendConversationThreadMutationPayload,
+      resolve: data => data,
     },
   },
   mutateAndGetPayload: ({ id, from, to, body_text }, request, { rootValue: { accessToken } }) => {
     if (!accessToken) return null
-    return gravity
-      .with(accessToken, { method: "POST" })("me/token", {
-        client_application_id: IMPULSE_APPLICATION_ID,
-      })
+    let impulseToken
+    return getImpulseToken(accessToken)
       .then(data => {
+        impulseToken = data.token
         return Promise.resolve(
```

Comment:
you don't need `Promise.resolve` here, you can just return `impulse.with(...)` since it's already returning a promise